pre-receive-hooks/block-outdated-clients.sh

```diff
@@ -0,0 +1,140 @@
+#!/user/bin/env bash
```

Comment:
Fix bash interpreter path 🙂 
```suggestion
#!/usr/bin/env bash
```